writings/2022-08-23-flashbots-protect-merge-beyond.mdx

```diff
@@ -0,0 +1,51 @@
+
+Flashbots will be a block builder and will use our builder to continue to offer the benefits of Flashbots Protect to users after the merge. However, mev-boost will go live after ~10 epochs have been finalized after the merge. As a result, there will be a 1 hour period of time where Flashbots Protect will be down. **Transactions sent to the Protect RPC during this 1 hour window after the merge will be dropped instantly and look like the following in MetaMask:**
+
+![dropped-tx-metamask.png](/img/dropped-tx-metamask.png)
+
+Furthermore, following the merge, there will be a stabilization period with many uncertainties—such as how fast validators will roll out mev-boost and how much competition there will be between builders. Due to these unknowns, we cannot determine beforehand how quickly the Flashbots builder will be able to include transactions, and therefore when we will turn on Flashbots Protect services. So, **Flashbots Protect will continue to drop transactions until we can guarantee transaction frontrunning and revert protection with a low time to inclusion**. We will announce the downtime and when we are live again through the[ #announcements channel in the Flashbots Discord](https://discord.gg/7hvTycdNcK) and[ various team Twitter](https://twitter.com/bertcmiller) accounts!
+
+To get your transaction included on-chain while Protect is down switch back your network to Ethereum Mainnet.
+
+
+## Post-merge Improvements
+
+Today [Flashbots Protect](https://docs.flashbots.net/flashbots-protect/overview) provides frontrunning protection, no cost for failed transactions, and priority in blocks, and [Fast Mode](https://docs.flashbots.net/flashbots-protect/rpc/fast-mode) provides only frontrunning protection for a faster time to inclusion. Post-merge, under the hood, we will add revert protection to Fast Mode and this will be the default for all transactions. In other words, **all transactions sent to Flashbots Protect RPC after the merge will have frontrunning and revert protection by default. **There are **no changes required** for users sending transactions to [https://rpc.flashbots.net](https://rpc.flashbots.net/fast) or [https://rpc.flashbots.net/fast](https://rpc.flashbots.net/fast). 
+
+But this is just the beginning. We are super excited by the cool UX features that being able to propose full blocks through mev-boost could enable—account abstraction, sponsored transactions, fee payment in popular ERC20 tokens, user MEV kickbacks, and more. Dapps and wallets who are interested in providing these features for your users, reach out!
```

Comment:
"enable—account abstraction" looks kind of weird, suggest you change to "enable, such as"